src/MasterPeace/Bundle/UpReadBundle/Controller/HomePageController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace MasterPeace\Bundle\UpReadBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+class HomePageController extends Controller
+{
+    /**
+     * @Route("/", name="homepage")
+     *
+     * @return Response
+     */
+    public function redirectAction()
+    {
+        $response = null;
```

Comment:
tas $response = null ,nereikia gal. Galima perrasyti taip:

        if ($this->getUser()->isStudent()) {
            return $this->render('MasterPeaceUpReadBundle:HomePage:student.html.twig');
        }

        return $this->render('MasterPeaceUpReadBundle:HomePage:teacher.html.twig');